alerts_intro.md

```diff
@@ -1,5 +1,15 @@
 ### What Is an Alert?
 
-An alert defines the conditions under which metric values indicate a system problem and set of targets to be notified when the condition evaluates to true or false for a specified period of time. When an alert changes state, notifications are sent to targets such as email addresses, pager services, and real-time communication channels.
+An alert defines the conditions under which metric values or logs data indicate a system problem when the condition evaluates to true or false for a specified period. 
 
-To disable checking alert conditions, you can snooze alerts for a fixed time window or put a set of alerts or sources in a custom [maintenance window](https://docs.wavefront.com/maintenance_windows_managing.html#creating-a-maintenance-window).
+**Manage Metrics Alerts**
+ 
+ When a metrics alert changes state, our service sends notifications to the targets your define, such as email addresses, pager services, and real-time communication channels.
+
+To pause the checking alert conditions, you can snooze alerts for a fixed time window or put a set of alerts or sources in a custom [maintenance window](https://docs.wavefront.com/maintenance_windows_managing.html#creating-a-maintenance-window).
+
+**Managing Logs Alerts**
+
+You can create logs alerts and see the firing events of a logs alert. If you have Logs permissions, you see the **Logs Alert** tab, which lists the logs alerts you create. If you have Logs and Alerts permissions, you can create and manage logs alerts. 
```

Comment:
If you have the **Logs** permission, you see the **Logs Alert** tab, which lists the logs alerts you create. If you have the **Logs** and **Alerts** permissions, you can create and manage logs alerts.